modules/caddyhttp/fileserver/browse.html

```diff
@@ -714,15 +719,21 @@
 		stroke: #ccc !important;
 	}
 }
+
+@media (prefers-color-scheme: dark) and (max-width: 600px) {
+	h1 {
+		color: #fff;
+	}
+}
```

Comment:
Nice catch, thanks. Why does this have to be limited to < 600px? I feel like the color shouldn't depend on screen size.

Maybe move it into the media query above? (And use `white` for consistency, not that it matters, just a nit.)

Reply:
Done.